python/pydantic_core/core_schema.py

```diff
@@ -3475,7 +3467,8 @@ def arguments_schema(
 
     Args:
         arguments: The arguments to use for the arguments schema
-        populate_by_name: Whether to populate by name
+        validate_by_name: Whether to populate by the parameter names, defaults to False.
+        validate_by_alias: Whether to populate by parameter aliases, defaults to True.
```

Comment:
```suggestion
        validate_by_name: Whether to populate by the parameter names, defaults to `False`.
        validate_by_alias: Whether to populate by the parameter aliases, defaults to `True`.
```

sorry typo from my previous suggestion